webapp/packages/core-blocks/src/Split/SplitControls.tsx

```diff
@@ -65,18 +66,22 @@ export const SplitControls: React.FC = function SplitControls() {
       onDoubleClick={split.state.onDoubleClick}
     >
       {split.state.mode !== 'minimize' && (
-        <button
-          className={s(styles, { button: true, primary: !inverse, resizeButton: isResizeMode })}
-          type="button"
+        <SplitControlButton
+          className={s(styles, { resizeButton: isResizeMode })}
+          mode={inverseMode}
+          split={split.state.split}
+          variant={!inverse ? 'primary' : undefined}
           onClick={handlers.handleCollapse}
-        >
-          <div className={s(styles, { ripple: true })} />
-        </button>
+        />
       )}
       {split.state.mode !== 'maximize' && (
-        <button className={s(styles, { button: true, primary: inverse, resizeButton: isResizeMode })} type="button" onClick={handlers.handleExpand}>
-          <div className={s(styles, { ripple: true })} />
-        </button>
+        <SplitControlButton
+          className={s(styles, { resizeButton: isResizeMode })}
+          mode={inverseMode}
+          split={split.state.split}
+          variant={inverse ? 'primary' : undefined}
```

Comment:
this is not the right usage of variant property. It relates to inner Button behaviour/appearance, but you are using it for adding a custom class and logic for SplitControlButton. It's better to add a special property.